docs/topics/downloader-middleware.rst

```diff
@@ -852,6 +852,11 @@ Default: ``2``
 
 Maximum number of times to retry, in addition to the first download.
 
+.. reqmeta:: max_retry_times
```

Comment:
This looks redundant with what's in `docs/topics/request-response.rst`. I suggest that you only reference the meta key here, something like:
```
Maximum number of retries can also be specified per-request using :reqmeta:`max_retry_times` blablabla
```
(You can rephrase this of course.)
There are other uses of `:reqmeta:` in this downloader-middleware.rst